src/dependency-manager/src/service-config/v1.ts

```diff
@@ -172,7 +177,13 @@ export class ServiceConfigV1 extends ServiceConfig {
   }
 
   getDebugOptions(): ServiceDebugOptions | undefined {
-    return this.debug ? { command: this.debug } : undefined;
+    if (this.debug || this.entrypoint) {
```

Comment:
Oh wait I had this wrong. For a sec I thought debug options was from the env json. You can probably undo this change. Could you add support for overriding the entrypoint in the env json? We currently have a debug block where you can override command.